css/index.css

```diff
@@ -105,27 +146,55 @@ p {
     width: 12.5%;
     height: 100px;
     background: black;
-    margin: 20px 2.5%;
+    margin: 2rem 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
+    font-size: 1.6rem;
+}
+
+.middle-content .boxes .box:nth-child(2) {
+    background: forestgreen;
```

Comment:
Take a closer look at how you are implementing the colors. None of the colors showed up and it just may be that you're calling it incorrectly! The simplest (and a little tedious) thing you can do, is to hardcode your html to say something like  <div class="box" id="uniqueIdYouCreate"> and calling each of the colors accordingly.